src/ray/core_worker/context.cc

```diff
@@ -56,7 +57,8 @@ struct WorkerThreadContext {
   /// Number of tasks that have been submitted from current task.
   int task_index_;
 
-  /// Number of objects that have been put from current task.
+  /// Number of return objects for the current task plus the number of objects that
+  /// have been put from current task.
```

Comment:
```suggestion
  /// Used to calculate ObjectIDs for put objects. The index starts at the number of return values from this task
  /// to reserve those indices for the return objects.
```

Reply:
Way better, thanks. This is almost word-for-word what I was going to update it to. 😁